src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/ProjectPage.xaml

```diff
@@ -308,6 +310,18 @@
 								   Style="{StaticResource Typo08}" />
 					</StackPanel>
 				</Grid>
+				
+				<!-- ComboBox Popup BG 1 -->
```

Comment:
What does this stand for?

Reply:
The idea is that to get the dark overlay from the ComboBox Popup I added an overlay that's only visible when IsDropDownOpen is true. I had to make 2 cause I can't bind to 2 object at the same time.